core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -2040,7 +2041,7 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
           // resources instead of synching an extra group. This helps to keep the number of google groups a user is in below
           // the limit of 2000
           Future.successful(())
-        } else if (WorkspaceAccessLevels.withPolicyName(policyName.value).isDefined) {
+        } else if (WorkspaceAccessLevels.withPolicyName(policyName.value).isDefined || policyName == SamWorkspacePolicyNames.canCompute) {
```

Comment:
fixes workspace/google-project creation -- workspace can-compute policy needs to be synced with google.